case_studies/galaxy_clustering/data_generation/DES_data_extraction.py

```diff
@@ -0,0 +1,48 @@
+# flake8: noqa
```

Comment:
flake8 was throwing an "import module pattern" warning on line 0 for some reason. Disabled the check for this file.